jsapp/js/components/activity/formActivity.tsx

```diff
@@ -88,25 +95,28 @@ export default function FormActivity() {
         </div>
       </div>
       <div className={styles.tableContainer}>
-        {detailsModalData &&
+        {detailsModalData && (
           <KoboModal
             isOpen
             size='medium'
             onRequestClose={() => setDetailsModalData(null)}
           >
-            <KoboModalHeader onRequestCloseByX={() => setDetailsModalData(null)}>
+            <KoboModalHeader
+              onRequestCloseByX={() => setDetailsModalData(null)}
+            >
               <ActivityMessage data={detailsModalData} />
             </KoboModalHeader>
 
             <section className={styles.detailsModalContent}>
               <pre>{JSON.stringify(detailsModalData, null, '  ')}</pre>
             </section>
           </KoboModal>
-        }
+        )}
 
         <PaginatedQueryUniversalTable<ActivityLogsItem>
           columns={columns}
           queryHook={useActivityLogsQuery}
+          queryHookData={queryData}
```

Comment:
Tested doing some actions and all of them appear successfuly. There is problem with `ActivityMessage` not supporting all of them (see other review comment)

Reply:
Added everything that was missing, now we have more keys than the BE. We will need to find a way to sync these keys.